docs/dev/development.adoc

```diff
@@ -181,7 +182,16 @@ $ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--ena
 
 *Integration tests:*
 
-Integration tests utilize link:https://github.com/onsi/ginkgo[`Ginkgo`] and it's preferred matcher library link:https://github.com/onsi/gomega[`Gomega`] which define sets of test cases (spec). As per ginkgo test file comprises specs and these test file are controlled by test suite. Test and test suite files are located in `tests/integration` directory and can be called using `.PHONY` within `makefile`. Integration tests validates and focuses on specific fields of odo functionality or individual commands. For example, `cmd_app_test.go` or `generic_test.go`.
+Integration tests utilize link:https://github.com/onsi/ginkgo[`Ginkgo`] and it's preferred matcher library link:https://github.com/onsi/gomega[`Gomega`] which define sets of test cases (spec). As per ginkgo test file comprises specs and these test file are controlled by test suite. 
+
+Test and test suite files are located in `tests/integration` directory and can be called using `make test-integration`. 
+
+To run ONE individual test, you can either:
+
+* Supply the name via command-line: `ginkgo -randomizeAllSpecs -slowSpecThreshold=120 -timeout 7200s -nodes=1 -progress -v tests/integration/ -F "When executing catalog list without component directory"`
+* Modify the `It` statement to `Fit` and run `ginkgo -randomizeAllSpecs -slowSpecThreshold=120 -timeout 7200s -nodes=1 -progress -v tests/integration/`
```

Comment:
You can make it simple and generic like
```
* Modify the `It` statement to `Fit` and run `ginkgo test_dir_path_of_spec_file`
+
----
For example:
ginkgo tests/integration/
----
+

As i mentioned above you can uses a common NOTE of ``` NOTE: To customize output use ginkgo flags. see `ginkgo -h` for more details``` for both the steps

Reply:
Verified locally
```
$ ginkgo tests/integration/
Running Suite: Integration Suite
================================
Random Seed: 1572507116
Will run 1 of 122 specs    --------------- Running only one spec
[...]
```